bundle/README.md

```diff
@@ -46,7 +46,7 @@ You have access to the OpenShift CLI (oc).
 
 1. Accept the defaults and click **Create**. This creates the Istio control plane.
 
-1. Click **Workloads** -> **Pods**. Verify that the pods were created. `Running` should appear in the **Status** column. If the pods were successfully created, then Istio is installed and ready for use. For more information, see the upstream [Istio documentation](https://istio.io/latest/docs/setup/platform-setup/openshift/).
+1. Click **Workloads** -> **Pods**. Verify that the pod was created. `Running` should appear in the **Status** column. If the pod was successfully created, then Istio is installed and ready for use. For more information, see the upstream [Istio documentation](https://istio.io/latest/docs/setup/platform-setup/openshift/).
```

Comment:
This change itself LGTM, but I wonder if we should inspect the `Istio` resource instead of the pod. It should have the `Ready` status...

Reply:
Good suggestion. I'll redeploy to walk through the steps, and then update the PR.

Reply:
Done